benchmarks/pug/index.ts

```diff
@@ -0,0 +1,124 @@
+import pug from 'pug';
+
+const templates = {
+  purchase: pug.compile(`
+.purchase.purchase-card
```

Comment:
I'm pretty sure this isn't the intended use for DX pug; please write it in a way that actually utilizes .pug files.